pandas/tests/groupby/test_function.py

```diff
@@ -573,6 +573,32 @@ def test_ops_general(op, targop):
     tm.assert_frame_equal(result, expected)
 
 
+def test_ops_not_as_index(reduction_func):
```

Comment:
these should be in test_grouper i think (search for tests that exercise as_index=False)